af/firefox-ios.xliff

```diff
@@ -2721,8 +2880,8 @@
     </header>
     <body>
       <trans-unit id="CFBundleDisplayName">
-        <source>Share To</source>
-        <target>Deel na</target>
+        <source>$(MOZ_BUNDLE_DISPLAY_NAME)</source>
+        <target>$(MOZ_BUNDLE_DISPLAY_NAME)</target>
```

Comment:
This is expected - this should not have been localized. Using `$(MOZ_BUNDLE_DISPLAY_NAME)` is the right thing to do here.